core/src/main/scala/io/finch/Outputs.scala

```diff
@@ -4,47 +4,66 @@ import com.twitter.finagle.http.Status
 
 trait Outputs {
 
-  // See https://gist.github.com/vkostyukov/32c84c0c01789425c29a to understand how this list is assembled.
+  // See https://gist.github.com/vkostyukov/32c84c0c01789425c29a to understand how this list
+  // is assembled.
 
   // 2xx
-  def Ok[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Ok)                                               // 200
-  def Created[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Created)                                     // 201
-  def Accepted[A](a: A): Output.Payload[A] = Output.Payload(a, Status.Accepted)                                   // 202
-  def NoContent[A](a: A): Output.Payload[A] = Output.Payload(a, Status.NoContent)                                 // 204
+  def Ok[A](a: A): Output[A] = Output.payload(a, Status.Ok)                                   // 200
+  def Created[A](a: A): Output[A] = Output.payload(a, Status.Created)                         // 201
+  val Accepted: Output[Unit] = Output.unit(Status.Accepted)                                   // 202
```

Comment:
Yeah, good question.

I was thinking this kind of information is better to provide in headers rather than a body since this is usually a URL pointing to some "monitor". Example: we create/post a new user, but it's an expensive process so we run it in a background. We could create an `Endpoint[Unit]` for that, not `Endopint[User]` indicating that there is no payload returned (which is true).